src/Common/src/System/Net/CookieParser.cs

```diff
@@ -614,7 +614,8 @@ internal Cookie Get()
                                     {
                                         domainSet = true;
                                         cookie.Domain = CheckQuoted(_tokenizer.Value);
-                                        cookie.IsQuotedDomain = _tokenizer.Quoted;
+                                        // TODO: this is internal.
+                                        // cookie.IsQuotedDomain = _tokenizer.Quoted;
```

Comment:
> I guess we can go with reflection option

That might break this for UWP.  Since we plan to take this managed listener implementation as the System.Net.HttpListener library for UWP, using reflection might not work the same.  In the past, reflection APIs weren't supported in UWP.  It might be better now....but something to investigate.